views/index.html

```diff
@@ -1,15 +1,15 @@
 <div class="pure-g full-height-block dark-bg splash">
     <a href="/team"><div class="splash-container"></div></a>
     <div class="pure-u-1 splash-tagline">
-      <span>dwyl is a <span class="tagline-highlight">team</span> of people using <span class="tagline-highlight">technology</span> to solve real problems</span>
+      <span>dwyl /dwɪɫ/ is a <span class="tagline-highlight">worldwide community</span> of people using <span class="tagline-highlight">technology</span> to solve real problems.</span>
```

Comment:
I do...not sure if other people do...this comes from https://github.com/dwyl/video/issues/24 and https://github.com/dwyl/hq/issues/249 - feel free to remove if we don't want it on there!